src/win/win-helpers.cpp

```diff
@@ -96,33 +96,33 @@ namespace librealsense
             if(tokens.size() < 1 || (tokens[0] != R"(\\?\usb)" && tokens[0] != R"(\\?\hid)")) return false; // Not a USB device
             if(tokens.size() < 3)
             {
-                LOG_ERROR("malformed usb device path: " << name);
+                LOG_WARNING( "malformed usb device path: " << name );
```

Comment:
Why is this warning rather than debug for all the rest?

Reply:
No special reaon, this seems to be that first sanity check that the connect device is really a USB/HID device.
And normally we dont see it
So I changed to warning